subsys/bluetooth/controller/Kconfig

```diff
@@ -242,5 +242,27 @@ config BT_CTLR_ECDH_IN_MPSL_WORK
 
 endif # BT_CTLR_ECDH
 
+if MPSL_CX
+choice BT_CTLR_CX_ADV_CONFIG
```

Comment:
I would be happy if you could prefix SoftDevice Controller specific Kconfig's with `BT_CTLR_SDC`. See `BT_CTLR_SDC_PERIODIC_SYNC_BUFFER_COUNT` and similar